### ISSUE TITLE
Javascript broken on IE11. Possibly due to use of `=>` syntax?

### ISSUE THREAD

**Issue by jrochkind** [projectblacklight/blacklight#2252]
Via Slack last week, @mejackreed suggested that Blacklight is expected to work on IE11 (since Bootstrap supports IE 10-11), and problems would be welcome as GH issues.

I reproduced this problem in a blank app following [Blacklight quickstart](https://github.com/projectblacklight/blacklight/wiki/Quickstart) using Blacklight 7.6.0 and Rails 5.2.4.1.  (I don't believe the Rails versions is relevant, but had trouble with BL quickstart and latest Rails 6 per #2250, so created a quickstart app with last Rails 5.2 instead -- which I believe is also supposed to be supported). 

Loading app home page in IE11, with developer tools console open, has these errors in the console:

```
SCRIPT5009: 'Blacklight' is undefined
localhost:3010 (152,3)
```

Clicking on that source line reveals:

```
<script>
  Blacklight.onLoad(function() {
```

Also, next in console:
```
SCRIPT1002: Syntax Error
blacklight.self-332392b9fae9e8f0882ea7d8f08d3013cb716e0e8c23caa73c63f7c431e3085b.js (102,71)
```

Clicking on that source line reveals this line:

```
  document.querySelectorAll('button.collapse-toggle').forEach(button => {
```

Which seems to be coming from [here](https://github.com/projectblacklight/blacklight/blob/24e316398c6617eb18c68fc8dfce36a8b62a4e9d/app/javascript/blacklight/button_focus.js) in original source file, or [here](https://github.com/projectblacklight/blacklight/blob/efc3ab97c68f2fcc6e9354dd2c036a6709d4ed96/app/assets/javascripts/blacklight/blacklight.js#L99-L107) in combined `blacklight.js` file. 

It may be that IE11 does not know the `=>` syntax?  It does not look like "arrow function" syntax is supported in IE11.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Browser_compatibility

It may be that that syntax error is responsible for the earlier error with `Blacklight` not being defined, the code couldn't be loaded because of the syntax error. 

While Bootstrap JS is still working, I believe this results in all (or at least some/most?) Blacklight-specific JS not being loaded in IE11 at all (due to the syntax error), thus not working. For instance, IE11 gets the non-JS "Bookmark" functionality, the button instead of the checkbox. 

While this may be a fine non-JS fallback, it also means that any JS you are including in the same `application.js` will not be loaded, due to the syntax error preventing load of the whole file. 

Current `demo.projectblacklight.org` also reports a `SCRIPT1002: Syntax Error` -- it's harder to see the source line, since it's leading to minified JS, but I think I see an arrow function in there, so probably same thing. For some reason current `demo.projectblacklight.org` does not report the `SCRIPT5009: 'Blacklight' is undefined` my quickstart app did. 

I believe this problem goes back further than BL 7.6.0. My own custom local app is using Blacklight 7.4.2 (with sprockets), and I believe exhibits the same problem. It may go back further than that as well.

**Comment by jcoyne** [comment]
@jrochkind are you using webpack?  This should transpile the code to whatever compatibility level you require.

**Comment by jrochkind** [comment]
No, I'm not using webpack for Blacklight assets btw.

**Comment by jcoyne** [comment]
Fixed by #2510
